API/NoDocs/Aspire.Azure.Security.KeyVault/AzureSecurityKeyVaultSettings.cs

```diff
@@ -0,0 +1,17 @@
+// Assembly 'Aspire.Azure.Security.KeyVault'
+
+using System;
+using System.Runtime.CompilerServices;
+using Aspire.Azure.Common;
+using Azure.Core;
+
+namespace Aspire.Azure.Security.KeyVault;
+
+public sealed class AzureSecurityKeyVaultSettings : IConnectionStringSettings
```

Comment:
Just to triple check one last time before GA - we're going to unify all the KV packages for Keys, Secrets, Certs, etc., into a single Aspire component?